src/interfaces/modular/shogun-base.i.jinja2

```diff
@@ -0,0 +1,20 @@
+/*
+ * THIS IS A GENERATED FILE!  DO NOT CHANGE THIS FILE!  CHANGE THE
```

Comment:
This is the Jinja template for the automatically generated file. This comment is in the template so that it also appears in the auto-generated file. I can also use python code to make this comment in the generated file instead of mentioning it in the template.